iron-flex-layout.html

```diff
@@ -10,6 +10,23 @@
 
 <link rel="import" href="../polymer/polymer.html">
 
+<script>
+/**
+The `<iron-flex-layout>` component provides simple ways to use [CSS flexible box layout](https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Flexible_boxes), also known as flexbox. This component provides two different ways to use flexbox:
+
+1. [Layout classes](https://github.com/PolymerElements/iron-flex-layout/tree/master/classes). The layout class stylesheet provides a simple set of class-based flexbox rules. Layout classes let you specify layout properties directly in markup.
+
+2. [Custom CSS mixins](https://github.com/PolymerElements/iron-flex-layout/blob/master/iron-flex-layout.html). The mixin stylesheet includes custom CSS mixins that can be applied inside a CSS rule using the `@apply` function.
+
+A complete [guide](https://elements.polymer-project.org/guides/flex-layout) to `<iron-flex-layout>` is available.
+
+@demo demo/index.html
+**/
+Polymer({
+  is:'iron-flex-layout'
```

Comment:
I think in this context creating a fake element so the docs show up may be even more lie-y than declaring it a fake behavior. However, it works, which the behavior won't unless we also add script.

I avoided adding script to my PR on the grounds that it added page weight (and unnecessarily blocks the parser if I understand correctly) in order to fix the docs. I still feel like that's the correct call in general, but this has been hanging fire for months now, which is kind of an issue.

So I guess as @notwaldorf said, we need @garlicnation to weigh in with an ETA on this fix.

Cheers,
Arthur